chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,47 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting *LC_ALL* variable is NOT recommended and can break Chef's parsing of command output in unexpected ways. Use one of the more specific *LC_* properties as needed.
```

Comment:
Setting the *LC_ALL* variable is NOT recommended. As a system-wide setting, *LANG* should provide the desired behavior. *LC_ALL* is intended to be used for temporarily troubleshooting issues rather than an every day system setting. Changing *LC_ALL* can break Chef's parsing of command output in unexpected ways. Use one of the more specific *LC_* properties as needed.